resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
I currently don't see much of a problem, but as you know I'm not very good at front-end stuff. I'd like @thervh70 to take a look at this :)